packages/swarm-components/src/Textarea.jsx

```diff
@@ -18,6 +18,10 @@ type Props = {
     /**
      * Value for textarea.
      */
+    name: string,
```

Comment:
This is a required prop for `Textarea`. The tests and examples in the docs need to be updated to supply a name. Otherwise, both the tests and the browser will produce errors.